Library/Homebrew/test/formula_spec.rb

```diff
@@ -727,7 +727,7 @@ def test
     stub_formula_loader(f1)
 
     python = PythonRequirement.new
-    x11 = X11Requirement.new("x11", [:recommended])
+    x11 = (OS.mac? ? X11Requirement : XorgRequirement).new("x11", [:recommended])
```

Comment:
should we move that `if OS.mac?` to `X11Requirement` which would require `XorgRequirement` on Linux platforms? This is a general question so it should not block this PR